src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/Mappers/BulkUploadMapper.cs

```diff
@@ -75,7 +76,7 @@ public async Task<IList<Apprenticeship>> MapFrom(long commitmentId, IEnumerable<
         private UploadRowErrorViewModel MapError(IGrouping<int?, UploadError> arg)
         {
             var firstRecord = arg.FirstOrDefault();
-            Func<string, string> stringOrDefault = str => string.IsNullOrWhiteSpace(str) ? "�" : str;
+            Func<string, string> stringOrDefault = str => string.IsNullOrWhiteSpace(str) ? "�" : str;
```

Comment:
This character seems like it could cause issues

Reply:
updated - not sure how that got in there!